packages/widgets/src/components/TradingInfoCard/TradingInfoCard.tsx

```diff
@@ -0,0 +1,65 @@
+import { css, useTheme } from '@emotion/react';
+import { CardBase, CardContent, Typography } from '@ssa-ui-kit/core';
+
+import TradingInfoCardContent from './TradingInfoCardContent';
+import TradingInfoCardTooltip from './TradingInfoCardTooltip';
+import { ITradingInfoCardProps } from './types';
+
+const TradingInfoCard = ({
+  value,
+  unit,
+  title,
+  onClick,
+  icon,
+}: ITradingInfoCardProps) => {
+  const theme = useTheme();
+  const number = Number(value);
+  const isInteger = Number.isInteger(number);
+  const currentValue = Number.isNaN(number) ? value : Math.floor(number);
+
+  return (
+    <CardBase
+      data-testid="card"
+      onClick={() => {
```

Comment:
We discussed adding the `role` attribute (and the others) when the `onClick` handler passed.

Reply:
Done